### PR TITLE
[1LP][RFR] Automating test user service request details

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_request.py::test_user_view_service_request_details --long-running -svv }}

- test_user_view_service_request_details

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 520dd3b80a658473f0921539bfb33bb77a2fef2c

The *local* fixture **`user`** is used in the following files:
- **cfme/tests/services/test_request.py**
    - *test_user_view_service_request_details*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/services/test_request.py [hunk 1]
```diff
@@ -71,3 +107,34 @@ def test_services_requester_dropdown_sorting(appliance, generic_catalog_item):
...
+            1.
+            2.
+            3.
+            4. User should be able to see the provision request
+    """
+    request = ServiceCatalogs(
+        appliance, catalog=generic_catalog_item.catalog, name=generic_catalog_item.name
+    ).order()
+    with user:
+        appliance.server.login(user)
```
Comment (valaparthvi) [inline cfme/tests/services/test_request.py:139]: If you're using `with user`, do you need to use `appliance.server.login(user)`, have you tried without it?

**Comment by dajoRH** [comment]
I detected some fixture changes in commit f520e1c5bf7ac20d055d77215b60f4b958d50d98

The *local* fixture **`user`** is used in the following files:
- **cfme/tests/services/test_request.py**
    - *test_user_view_service_request_details*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/services/test_request.py (hunk 1 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_request.py]: Yes right, we can log in without  `appliance.server.login(user)` thanks

**Pull request merged**